src/lib.rs

```diff
@@ -611,37 +578,37 @@ impl Client {
     pub fn build_report(&self) -> ReportBuilder {
         ReportBuilder {
             client: self,
-            send_strategy: None,
         }
     }
 
-    /// Function used internally to send payloads to Rollbar as default `send_strategy`.
-    fn send(&self, payload: String) -> thread::JoinHandle<Option<ResponseStatus>> {
+    /// Function used internally to send payloads to Rollbar.
+    async fn send(&self, payload: String) -> Option<ResponseStatus> {
+        let payload_clone = payload.clone();
         let body = hyper::Body::from(payload);
```

Comment:
payload is consumed here, that is why i had to clone it first in order to print it out later if there was a problem